### ISSUE TITLE
Check on CI that docs.rs build is successfull

### ISSUE THREAD

**Issue by hug-dev** [parallaxsecond/parsec#369]
It's not ideal to make a release and hope for the best that the docs.rs build passes 😬 
We should add on CI a way to check that the compilation works there. Maybe there is a Docker container that they use? Or someway to reproduce their environment.

**Comment by ionut-arm** [comment]
Yeah, it's annoying, I'll have a quick look - from what I remember, they have this massive local storage with all the crates and don't need to go over the network, and that would take a loooong time to build.

**Comment by ionut-arm** [comment]
They describe [here](https://github.com/rust-lang/docs.rs/blob/master/README.md#build-subcommand) how to check it locally. The first time around it'll take a while until it builds the environment, but after that it's going to be quick because everything is cached. You do need `docker-compose`, though!

I'll add this to the "publishing a crate" part of the book.
